### PR TITLE
Added wrappers for libleptonica and libtesseract for cob_read_text

### PR DESCRIPTION
**Pull request by ipa-rmb**

### CODE REVIEW

File: libtesseract/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libtesseract)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS
+	libleptonica
```
Comment (ipa-nhg) [inline libtesseract/CMakeLists.txt]: @ipa-rmb shouldn't be `libleptonica-dev` https://github.com/ros/rosdistro/blob/09d3b8fba89403c173d55ed0f48863e18eda6f60/rosdep/base.yaml#L1851 ?

**Comment by floweisshardt** [comment]
travis reports the following while building libtesseract:

```
checking for leptonica... configure: error: leptonica not found
make[3]: *** [installed] Error 1
make[2]: *** [cob_extern/libtesseract/CMakeFiles/build_tesseract] Error 2
make[1]: *** [cob_extern/libtesseract/CMakeFiles/build_tesseract.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

@ipa-rmb: seems as there is a build dependency between tesseract and leptonica which is not properly reflected in package.xml and/or CMakeLists.txt

**Comment by ipa-rmb** [comment]
Yes, I figured this our as well. I am working on it...

**Comment by fmessmer** [comment]
I will have a look at this after current issues with the changelogs are resolved (see #68), i.e. I will rebase @ipa-rmb's branch to "forthcoming" tag ```0.6.10```, i.e. these new libraries will be released in ```0.6.11```

**Comment by fmessmer** [comment]
rebased version see #70 
@ipa-rmb please proceed further development/testing from my branch [ipa-fxm/new_wrappers](https://github.com/ipa-fxm/cob_extern/tree/new_wrappers)

**Comment by fmessmer** [comment]
Do not merge!

See my https://github.com/ipa320/cob_extern/pull/70#issuecomment-287335590

**Comment by fmessmer** [comment]
tested the system dependencies with @ipa-rmb 
seems to work with minor adjustments to `cob_object_perception/cob_read_text`
@ipa-rmb will cleanup and commit them

**Pull request closed without merging**